### PR TITLE
AC Version Updates

### PR DESCRIPTION
**Pull request by jwitz**
This PR includes both changes for AC 2.2.0 and 2.2.1 as well as some facelifts for other AC-related content.

### CODE REVIEW

File: enterprise/v0.23/08_astronomer-certified/01_image-architecture.md [hunk 1]
```diff
@@ -27,40 +27,41 @@ The Astronomer Certified Docker image is built from the Python wheel and incorpo
...
 
 ## Image Requirements
 
 Running Airflow with the Astronomer Certified Docker image requires specific versions for key system components.  
 
-- Python: 3.6, 3.7, 3.8
+- Python: 3.6, 3.7, 3.8, 3.9
 - Database: PostgreSQL (11, 12), MySQL (5.7, 8.0+)
-- System Distribution: Debian 10 (Buster)
+- System Distribution: Debian 10 (Buster) for AC <=v2.1.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```
Comment (kaxil) [inline enterprise/v0.23/08_astronomer-certified/01_image-architecture.md:46]:
```suggestion
- System Distribution: Debian 10 (Buster) for AC <v2.2.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```

As we have 2.1.3 and 2.1.4 too

File: enterprise/v0.23/08_astronomer-certified/01_image-architecture.md [hunk 2]
```diff
@@ -27,40 +27,41 @@ The Astronomer Certified Docker image is built from the Python wheel and incorpo
...
+> **Note:** While the Astronomer Certified Python Wheel supports Python versions 3.6, 3.7, and 3.8, Astronomer Certified Docker images have been tested and built only with Python 3.7. To run Astronomer Certified on Docker with Python 3.6 or 3.8, you can create a custom image with a different Python version specified. For more information, read [Change Python Versions](https://www.astronomer.io/docs/enterprise/v0.25/develop/customize-image#build-with-a-different-python-version).
 
 These requirements are slightly different for running only the Python wheel. For the Python wheel, you can use:
 
-- Python: 3.6, 3.7, 3.8
+- Python: 3.6, 3.7, 3.8, 3.9
 - Database: PostgreSQL (9.6, 10, 11, 12, 13), MySQL (5.7, 8+), SQLite (3.15.0+)
-- System Distribution: Debian 10 (Buster)
+- System Distribution: Debian 10 (Buster) for AC <=v2.1.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```
Comment (kaxil) [inline enterprise/v0.23/08_astronomer-certified/01_image-architecture.md:54]: Same as above

File: enterprise/v0.25/08_astronomer-certified/01_image-architecture.md [hunk 3]
```diff
@@ -27,30 +27,31 @@ The Astronomer Certified Docker image is built from the Python wheel and incorpo
...
 
 > **Note:** While the Astronomer Certified Python Wheel supports Python versions 3.6, 3.7, and 3.8, Astronomer Certified Docker images have been tested and built only with Python 3.7. To run Astronomer Certified on Docker with Python 3.6 or 3.8, you can create a custom image with a different Python version specified. For more information, read [Change Python Versions](https://www.astronomer.io/docs/enterprise/v0.25/develop/customize-image#build-with-a-different-python-version).
 
 These requirements are slightly different for running only the Python wheel. For the Python wheel, you can use:
 
-- Python: 3.6, 3.7, 3.8
+- Python: 3.6, 3.7, 3.8, 3.9
 - Database: PostgreSQL (9.6, 10, 11, 12, 13), MySQL (5.7, 8+), SQLite (3.15.0+)
-- System Distribution: Debian 10 (Buster)
+- System Distribution: Debian 10 (Buster) for AC <=v2.1.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```
Comment (kaxil) [inline enterprise/v0.25/08_astronomer-certified/01_image-architecture.md:54]: And here

**Review by kaxil — COMMENTED** [review]
Minor suggestions otherwise LGTM

File: enterprise/v0.23/08_astronomer-certified/01_image-architecture.md (hunk 2 above)
Comment (jwitz) [inline enterprise/v0.23/08_astronomer-certified/01_image-architecture.md:54]:
```suggestion
- System Distribution: Debian 10 (Buster) for AC <v2.2.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```

File: enterprise/v0.25/08_astronomer-certified/01_image-architecture.md (hunk 3 above)
Comment (kaxil) [reply enterprise/v0.25/08_astronomer-certified/01_image-architecture.md:54]:
```suggestion
- System Distribution: Debian 10 (Buster) for AC <v2.2.0 / Debian 11 (Bullseye) for AC >=v2.2.0
```

**Review by kaxil — APPROVED** [review]
Single suggestion

**Review by kaxil — APPROVED** [review]

**Pull request merged**